Autocoders/Python/src/fprime_ac/generators/templates/test/cpp.tmpl

```diff
@@ -117,7 +117,7 @@ $emit_cpp_params([ $param_maxHistorySize ])
 $emit_cpp_params([ $param_instance ])
     )
   {
-#for $ids, $prm_name, $type, $oset, $osave, $size, $default, $comment, $typeinfo in $parameters:
+#for $ids, $prm_name, $type, $osets, $osaves, $size, $default, $comment, $typeinfo in $parameters:
```

Comment:
As with `$osave` / `$osavs`, there's `$oset` / `$osets`. I have no idea if it's the same.

Reply:
This is good too. Yes, it is the same as above and should be plural.